CHANGELOG.md

```diff
@@ -92,6 +92,7 @@
 -   Adds Live and timed auctions in home view - luc
 -   Tab bar text now aligned and centered correctly - luc
 -   Adds iPad support for Auctions view layout - luc
+-   Bufixes for #743 #745 #768 #775 - chris
```

Comment:
Looking over the changelog I'm unsure of how to order ongoing changes / fixes according to the various sections listed. There's `dev`, `home`, `betas`, etc. Does the person who makes the release move items from one area to the other or...?

Reply:
The top two lines of the CHANGELOG: 

> // Please add your own contribution below inside the Master section
> // These docs are aimed at developers, but are shown inside collector GMV's channel

We add new release notes into the master section at the top, and describe the fixes rather than linking to an issue. You can make a one liner to describe what's changed since the last Emission release.

Reply:
The top comment seemed a bit confusing in light of the various additional sections (home, etc) so I assumed it was out of date. Will update 👍

Reply:
Yeah, those sections were because it took 2-3 months to make a single Emission release once, it became too much to keep track of as a single set (merge conflicts, mainly)